src/migrations/1762700000000-userAuthIdBackfill.ts

```diff
@@ -0,0 +1,222 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import {
+  createKratosIdentityFetcher,
+  MigrationKratosIdentityFetcher,
+} from './utils/kratos.identity.fetcher';
+import {
+  createMigrationAssignAuthIdHelper,
+  MigrationAssignAuthIdResult,
+} from '@domain/community/user/user.service';
```

Comment:
absolutely no linkage from migrations back into the business logic; that isolation is a must.

Reply:
this is one-time toy. Once we migrate, then we remove this code completely, it does not need to stay in the codebase. It will remain just pollution, if stay.

Reply:
In general, we have to separate long term efforts from one-time toy. No need to complicate things and focus on "purity", if code will be removed within 24 hours after migration. Absolute structural integrity isn't a target here. Target here is to make sure it works once but properly and reliable. It does.

Reply:
That's not purity, that's just not going to work and we won't understand it in 2 weeks time if an issue arises,  I am with @techsmyth. That's why when discussing on Friday we were discussing a one-off migration via **API**, not via typeorm - the selected approach is a massive overhead and breaks a ton of core principles + copy / pastes existing kratos logic creating duplicate code paths that add testing complexity. The whole typeorm path, testing migration path, is absolutely not needed here imo...